environment-api/src/main/java/com/sequenceiq/environment/api/v1/credential/model/CredentialBase.java

```diff
@@ -29,7 +29,6 @@
     @Size(max = 100, min = 5, message = "The length of the credential's name has to be in range of 5 to 100")
     @Pattern(regexp = "(^[a-z][-a-z0-9]*[a-z0-9]$)",
             message = "The name of the credential can only contain lowercase alphanumeric characters and hyphens and has start with an alphanumeric character")
-    @NotNull
```

Comment:
Hm, the env cred cannot be null

Reply:
the name can be https://github.com/hortonworks/cloudbreak/pull/8887/files?file-filters%5B%5D=#diff-592e562d2b48995f6e0279a1ef5606acR106